APMrover2/Rover.h

```diff
@@ -107,6 +107,10 @@
 #include "AP_Rally.h"
 #include "RC_Channel.h"                  // RC Channel Library
 
+#if LTM_TELEM_ENABLED == ENABLED
```

Comment:
It would be better to not have `LTM_TELEM_ENABLED` here but hide it behind HAL_MINIMIZE_FEATURES in the base library.

I'd prefer not to see this in the vehicle code.

I know you've made these changes based on comments here, but I've just chatted with tridge and he agrees it shouldn't touch the vehicle code.  I'll make a follow-up PR to move it back out into the GCS code (protected by `HAL_MINIMIZE_FEATURES` after this is merged - assuming you don't want to do it yourself, of course.

Reply:
I understand, and I'm leaving it on you Peter. I can see there one more little "issue" - you would probably need to add those dummy stuff connected to AP_LTM_Telem to vehicles that don't support LTM again (tracker, sub, tools/reply).